tests/rptest/services/redpanda.py

```diff
@@ -112,6 +121,38 @@ def start_node(self, node, override_cfg_params=None):
                 "Redpanda didn't finish startup in {RedpandaService.READY_TIMEOUT_SEC} seconds",
             )
 
+    def coproc_enabled(self):
+        coproc = self._extra_rp_conf.get('enable_coproc')
+        dev_mode = self._extra_rp_conf.get('developer_mode')
+        return coproc is True and dev_mode is True
+
+    def start_wasm_engine(self, node):
+        node_cmd = "%s/bin/node" % self.find_wasm_root()
```

Comment:
Is ducktape still running on Python 2 @ivotron ? 
@graphcareful if it's on Python 3 would you consider changing these to f-strings pretty please? 🥺 They're way easier to read